spec/controllers/wrangling_guidelines_controller_spec.rb

```diff
@@ -212,15 +308,31 @@
       it_redirects_to_with_notice(root_path, "I'm sorry, only an admin can look at that area")
     end
 
-    context "when logged in as admin" do
-      let(:guideline) { create(:wrangling_guideline) }
+    %w[board communications translation policy_and_abuse docs support open_doors].each do |role|
+      context "when logged in as an admin with #{role} role" do 
+        let(:guideline) { create(:wrangling_guideline) }
+        let(:admin) { create(:admin, roles: [role]) }
+        before { fake_login_admin(admin) }
+
+        it "redirects with error" do 
+          delete :destroy, params: { id: guideline.id }
 
-      before { fake_login_admin(admin) }
+          it_redirects_to_with_error(root_url, "Sorry, only an authorized admin can access the page you were trying to reach.")
+        end
+      end
+    end
 
-      it "deletes and redirects to index" do
-        delete :destroy, params: { id: guideline.id }
-        expect(WranglingGuideline.find_by_id(guideline.id)).to be_nil
-        it_redirects_to_with_notice(wrangling_guidelines_path, "Wrangling Guideline was successfully deleted.")
+    %w[tag_wrangling superadmin].each do |role| 
+      context "when logged in as an admin with #{role} role" do 
+        let(:guideline) { create(:wrangling_guideline) }
+        let(:admin) { create(:admin, roles: [role]) }
+        before { fake_login_admin(admin) }
+
+        it "deletes and redirects to index" do
+          delete :destroy, params: { id: guideline.id }
+          expect(WranglingGuideline.find_by_id(guideline.id)).to be_nil
```

Comment:
won't change

Reply:
No, it doesn't. `find(x)` would break the test as it's written, but Hound is suggesting using `find_by(id: x)`, not `find(x)`. And `find_by_id(x)` calls `find_by(id: x)` under the hood.

Reply:
oh, I see, thanks for the clarification